korman/exporter/animation.py

```diff
@@ -89,21 +90,23 @@ def get_ranges(*args, **kwargs):
                 if i is not None:
                     yield i.frame_range[index]
         atcanim.name = "(Entire Animation)"
+        sdl_name = bo.plasma_modifiers.animation.obj_sdl_anim
         atcanim.start = self._convert_frame_time(min(get_ranges(obj_action, data_action, index=0)))
         atcanim.end = self._convert_frame_time(max(get_ranges(obj_action, data_action, index=1)))
-
+        if isinstance(atcanim, plAgeGlobalAnim):
+            atcanim.globalVarName = anim_mod.obj_sdl_anim
+        if isinstance(atcanim, plATCAnim):
         # Marker points
```

Comment:
```suggestion
            # Marker points
```

Style nitpick: we want to make sure our indentation is correct :wink: